packages/host/app/components/operator-mode/detail-panel.gts

```diff
@@ -305,3 +394,11 @@ export default class DetailPanel extends Component<Signature> {
     </style>
   </template>
 }
+
+const resolveElementName = (el: Element) => {
+  let localName: string | undefined = el.localName;
+  if (isCardOrFieldElement(el)) {
+    localName = el.cardOrField.displayName;
+  }
+  return localName ?? '??';
```

Comment:
In which case will the `localName` be undefined? I suppose "??" should be replaced with something more descriptive

Reply:
Thanks. I put [No Name Found]. This place is going to change drastically when I make changes to names in follow up PR. But for now I think [No Name Found] is fine